tests/framework/user_guide/ravenTutorial/RomLoad.xml

```diff
@@ -23,11 +23,7 @@
   </Files>
 
   <Models>
-    <ROM name="rom" subType="NDinvDistWeight">
-      <Features>sigma-A,decay-A</Features>
-      <Target>A, time</Target>
-      <p>3</p>
-    </ROM>
+    <ROM name="rom" subType="pickledROM" />
```

Comment:
The documentation in user_guide/ravenRomTrainer.tex needs to be update since it still has:
```
In this example, the subtype \xmlString{NDinvDistWeight} of \xmlNode{ROM} is used instead of \xmlString{pickledROM},
since the subtype of ROM is already known.
\xmlExample{framework/user_guide/ravenTutorial/RomLoad.xml}{Models}
```

Reply:
very good catch... I fixed!